File: Python/String/split function/Truncate Sentence.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        thislist=s.split()
        ans=""
        i=0
        while k!=0:
            ans+=thislist[i]+" "
            i+=1
            k-=1
        
        return ans.strip() # remove trailing space

File: Java/String/split_function/Truncate Sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        // int word_cnt=0;
        String[] str=s.split(" ");
        StringBuilder ans = new StringBuilder();

        for (int i = 0; i < k; i++) {
            ans.append(str[i]).append(" ");
        }
        return ans.toString().trim();
    }
}
